src/Middleware/RequestThrottling/src/RequestThrottlingOptions.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.RequestThrottling
+{
+    public class RequestThrottlingOptions
+    {
+        /// <summary>
+        /// Maximum number of concurrent requests. Any extras will be queued on the server. 
```

Comment:
Do we usually document the value of the default value? I think we do in some places, and I rather like it.

Reply:
Agreed, let's put the default value in.

Reply:
I've also started to avoid using `<remarks>` tags for this. Otherwise, the default values will only show up on the MSDN page specific to the property and not in the class listing. For example with KestrelServerOptions.NoDelay, you can see the default value of true in the remarks section of the [MSDN NoDelay page](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.kestrelserveroptions.nodelay?view=aspnetcore-1.1#Microsoft_AspNetCore_Server_Kestrel_KestrelServerOptions_NoDelay), but the default value doesn't show up on the [MSDN KestrelServerOptions page](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.kestrelserveroptions?view=aspnetcore-1.1) that more people look at.

We might want to look at moving the existing `<remarks>` doc comments to the `<summary>` section en masse.

Reply:
I see this was resolved, but I don't see the default value in the doc comment.